VirtusLab-Open-Source/strapi-plugin-navigation
# feat: populate relations [CU-257cnmv]

## Ticket

https://github.com/VirtusLab/strapi-plugin-navigation/issues/147

## Summary

Add the possibility to configure which relations should be populated in API responses. The population is configured per content type through the plugin's Settings page.

## Test Plan

- Create content type with relation attribute
- Configure attribute to be populated
- Test render method and check if API response is correct.
